### PR TITLE
Port PerformanceMetrics to React

### PR DESCRIPTION
**Pull request by gettinToasty**

### CODE REVIEW

**Comment by andycreeth** [comment]
image
perfectly-balanced-as-all-things-should-be

File: app/components-react/shared/PerformanceMetrics.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+export default function PerformanceMetrics(props: { mode: TPerformanceMetricsMode }) {
+  const { CustomizationService, PerformanceService } = Services;
+
+  const v = useVuex(() => ({
+    pinnedStats: CustomizationService.views.pinnedStatistics,
+    cpuPercent: PerformanceService.views.cpuPercent,
+    frameRate: PerformanceService.views.frameRate,
+    droppedFrames: PerformanceService.views.droppedFrames,
+    percentDropped: PerformanceService.views.percentDropped,
+    bandwidth: PerformanceService.views.bandwidth,
```
Comment (holiber) [inline app/components-react/shared/PerformanceMetrics.tsx:20]: Just a nitpick but could be simplified
```
{
   pinnedStats: CustomizationService.views.pinnedStatistics
  ...pick(PerformanceService.views, 'cpuPrecent', 'frameRate', 'droppedFrames', 'precentDropped', 'bandwidth')
}
```

File: app/components-react/shared/PerformanceMetrics.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        return (
+          <span
+            key={attribute}
+            className={cx(
+              styles.performanceMetricWrapper,
+              classForStat(attribute),
+              'performance-metric-wrapper',
+            )}
+            onClick={() => updatePinnedStats(attribute, !v.pinnedStats[attribute])}
+            // v-tooltip="pinTooltip('CPU')"
```
Comment (holiber) [inline app/components-react/shared/PerformanceMetrics.tsx:78]: For react components you need a tooltip from `antd` library

**Review by andycreeth — APPROVED** [review]
LGTM

**Pull request merged**